libpages/config/config_v2.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2018 Keybase Inc. All rights reserved.
```

Comment:
This is pretty much the same as `config_v1.go`, except that the authentication logic changes to use SHA256.

Reply:
Is there no way to just extend the `V1` class (say by embedding it) and override just the `init` and `Authenticate` functions?